cl/people_db/filters.py

```diff
@@ -284,6 +313,19 @@ class AttorneyFilter(NoEmptyFilterSet):
         queryset=Party.objects.all(),
         distinct=True,
     )
+    filter_nested_results = filters.BooleanFilter(
+        field_name="roles", method="filter_roles"
+    )
+
+    # Attributes for the mixin
+    # **Important:** Keep this mapping up-to-date with any changes to
+    # RelatedFilters or custom labels in this class to avoid unexpected
+    # behavior.
+    join_table_cleanup_mapping = {
+        "parties_represented__docket": "docket",
+        "parties_represented__attorney": "attorney",
+        "parties_represented": "parties",
```

Comment:
I think the value in this line should be `party` ?

```suggestion
        "parties_represented": "party",
```
Otherwise I noticed that a filter like: `?parties_represented=8&filter_nested_results=true`

Raise: 
`Cannot resolve keyword 'parties' into field. Choices are: attorney, attorney_id, date_action, docket, docket_id, id, party, party_id, role, role_raw`

Reply:
Thanks for catching this.